packages/panels/resources/views/resources/pages/list-records.blade.php

```diff
@@ -6,27 +6,37 @@
 >
     <div class="flex flex-col gap-y-4">
         @if (count($tabs = $this->getTabs()))
-            <div
-                class="mx-auto max-w-full rounded-xl bg-white p-2 shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10"
-            >
-                <x-filament::tabs class="self-center">
-                    @foreach ($tabs as $tabKey => $tab)
-                        @php
-                            $activeTab = strval($activeTab);
-                            $tabKey = strval($tabKey);
-                        @endphp
-
-                        <x-filament::tabs.item
-                            :active="$activeTab === $tabKey"
-                            :badge="$tab->getBadge()"
-                            :icon="$tab->getIcon()"
-                            :icon-position="$tab->getIconPosition()"
-                            :wire:click="'$set(\'activeTab\', ' . (filled($tabKey) ? ('\'' . $tabKey . '\'') : 'null') . ')'"
-                        >
-                            {{ $tab->getLabel() ?? $this->generateTabLabel($tabKey) }}
-                        </x-filament::tabs.item>
-                    @endforeach
-                </x-filament::tabs>
+            <div class="flex flex-col max-w-full gap-4 mx-auto md:flex-row">
```

Comment:
Hmm, why is it flex-col on mobile, @saade @danharrin?

Reply:
Probably so the stuff in the render hooks gets added above the tabs instead of below

Reply:
Because of the before and after content. On mobile, the tabs should be w-full hence no space for the before and after hooks to render in the same place it was on the desktop.